### DISCUSSION TITLE
Keycloak HA Autoscaling Setup Configuration

### DISCUSSION THREAD

**Discussion by khamkarp** [keycloak/keycloak#24185] [Q&A]
Hi All, 

I am trying to setup autoscaling for KeyCloak HA setup (with infinispan cache, not external/remote) configuration on our Kubernetes Cluster which can be autoscaled based on metrics like CPU or Memory usage. 

Our current version of keycloak is `15.0.0` and we could see that there are lot of memory leak issues due to which memory usage is huge most of the time which makes it difficult to autoscale during the issue time.

Is there any specific architecture which we can use to acheive this autoscaling? 

Thanks in advance.

**Comment by vmuzikar** [comment]
@khamkarp Version 15 is quite old and not supported for a long time already. Could you please try it with the latest version?
